### ISSUE TITLE
false report of reportUntypedFunctionDecorator on pytest decorator?

### ISSUE THREAD

**Issue by paxcodes** [microsoft/pyright#1434]
**Describe the bug**
I get a `reportUntypedFunctionDecorator` when using a pytest decorator that [already has typing](https://github.com/pytest-dev/pytest/issues/7469#issuecomment-769449449).

**To Reproduce**
- Install pytest
- Have the following code

```python
from pytest import mark

@mark.usefixtures("createCampaign")
def test_it_can_schedule_an_email(self):
  pass
```
Pyright reports that the line that begins with `@mark ...` is violating "reportUntypedFunctionDecorator"

> Untyped function decorator obscures type of function; ignoring decorator

**Expected behavior**
It should not report the error because it is typed as the [pytest contributor says](https://github.com/pytest-dev/pytest/issues/7469#issuecomment-769449449).

**Comment by bluetech** [comment]
Hi @paxcodes, are you using the latest pytest? See #1187/pytest-dev/pytest#8053

**Comment by paxcodes** [comment]
@bluetech oh, how embarassing. I am not. Thanks! I'll close once I've confirmed that the update fixes it.

**Comment by erictraut** [comment]
I confirmed that it no longer happens with the latest version of pytest.
